makefile

```diff
@@ -2,6 +2,7 @@ P="\\033[32m[+]\\033[0m"
 
 MONGODB?=mongodb://localhost:27017/plate 
 KEYSTONE_DEV=true
+SOCKETIO_PORT ?= 3030
```

Comment:
The space around the `?=` will be ignored by the compiler.
Ref: https://www.gnu.org/software/make/manual/make.html#Setting
But I'll remove them for style consistency.